core/src/sigverify_stage.rs

```diff
@@ -84,6 +84,8 @@ struct SigVerifierStats {
     total_batches: usize,
     total_packets: usize,
     total_dedup: usize,
+    forwarded_packets: usize,
+    forwarded_dedup: usize,
```

Comment:
```suggestion
    total_forwarded_packets: usize,
    total_forwarded_dedup: usize,
```
What do you think about adding `total` to these? And if you change them here, remember to change in the datapoint submission too